swan-lake/resources/featured-scenarios/write-a-restful-api-with-ballerina.md

```diff
@@ -127,12 +211,45 @@ service /covid/status on new http:Listener(9000) {
 In this code:
 
 - Unlike normal functions, resource methods can have accessors. In this case, the accessor is set to `get`, which means only HTTP `GET` requests could hit this resource. Ballerina automatically serializes Ballerina records as JSON and sends them over the wire. 
-- The default HTTP response status code for a resource method other than `post` is `200 OK`. For an HTTP `POST` resource, the default HTTP response status code is `201 Created`. 
+- The default HTTP response status code for a resource method other than `post` is `200 OK`. The default HTTP response status code for an HTTP `POST` resource is `201 Created`. 
 
 ### Create the second resource to add data
 
-To create the second resource of the first endpoint to add new COVID-19 data to the dataset by ISO code, add the code below to the API template file (i.e., `service.bal`).
+Before creating the second resource, you need to create the records of the custom error types, as shown below.
+
+#### Define the error detail records
+
+You need to define the error detail records below for the second resource of the first endpoint.
```

Comment:
Let's avoid terms like error detail and use something like conflict response payload.

Reply:
Updated.